examples/ccl_sample_power.c

```diff
@@ -1,7 +1,7 @@
-#include "ccl.h"
 #include <math.h>
 #include <stdio.h>
-#include "ccl_params.h"
+#include <ccl_params.h>
+#include <ccl.h>
```

Comment:
Ditto on the order.

Reply:
I left two files in different order to show that the order wasn't important. Such that when one day it becomes important one of the case will fail.

Reply:
Hmmmm. We should prob just specify a single order and stick with it. I don't think anyone will remember why this was done. 

Also, why are there two header files? Can't we just have a single ccl.h?

Reply:
Either would be an API change How much are people concerned about this yet? I don't think this PR shall introduce API changes. I can switch to the same ordering in the examples for now.

Reply:
My understanding is that api changes are still ok but won’t be post v1